rpcs3/rpcs3qt/user_manager_dialog.cpp

```diff
@@ -1,13 +1,14 @@
-﻿#include "user_manager_dialog.h"
+﻿#include <QRegExpValidator>
+#include <QInputDialog>
+#include <QKeyEvent>
```

Comment:
I don't think both are needed

Reply:
They point to different headers and I had an undefined KeyRelease without both.